primitives/sgx/crypto/Cargo.toml

```diff
@@ -13,9 +13,13 @@ derive_more = { version = "0.99.5" }
 log = { version = "0.4.14", default-features = false }
 
 # sgx deps
+sgx_types = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
```

Comment:
Is this import deliberately not optional?

Reply:
Yes, this provides the `sgx_status_t` type, which is part of the `Error` and therefore also needed in std.

This lib has no conflicts with std.